docs/docs/sample-scripts/teams/create-team-and-add-members-and-owners.md

```diff
@@ -0,0 +1,81 @@
+# Create a Microsoft Teams team and bulk add members from CSV file
+
+Inspired by: [Rakesh Pandey](https://www.flexmind.co/blog/how-to-add-bulk-users-from-csv-file-to-ms-teams-using-powershell/), [Patrick Lamber](https://www.nubo.eu/Create-A-New-Microsoft-Team-And-Bulk-Add-Members-From-CSV-File/)
+
+This sample script shows you how to create a Team and add members and owners using a CSV.
+
+```powershell tab="PowerShell Core"
+# This script provisions a Group with owners and members and Teamifies it
+# The owners and members can be specified using a CSV file following this format
+## upn,type
+## pla@expertsinside.com,owner
+## eba@expertsinside.com,member
+## szu@expertsinside.com,member
+# The CLI will provision the Group adding the current user as owner. You can remove this user from the owners list by using the $removeYourSelfFromOwners parameter
+
+$fileExportPath = "<putyourcsvhere.csv>"
```

Comment:
Since we're not exporting anything, but rather loading the list of users to add, let's rename this variable to clearly denote its purpose

Reply:
I am going to do it as soon as possible.